### ISSUE TITLE
The child_parent_sku_relationship property is not coming from ProductTypeDefinitions 

### ISSUE THREAD

**Issue by FloderCC** [amzn/selling-partner-api-docs#936]
Good Morning,
I have tried calls to the ProductTypeDefinitions endpoint with various ProductType values such as LUGGAGE, and in all cases it does not return the child_parent_sku_relationship property required to specify the parent sku. It always returns me:
variations': {'description': 'Variations that product will use',
                                               'propertyNames': ['parentage_level',
                                                                 'variation_theme'],
                                               'title': 'Variations'}},
I would like to know if I can still send the value of child_parent_sku_relationship later or what is happening.
Thank you!

**Comment by chapmanjw** [comment]
@FloderCC Configuring the variation relationships between SKUs is not yet supported by the Listings Item APIs. This functionality will be enabled in a future release of the APIs. We will post an announcement when this functionality is available.

**Comment by MST1122** [comment]
@FloderCC are you using FeedAPI to create new Listing also? if yes, then how are generating forms for input from users based on product type?

**Comment by FloderCC** [comment]
> 
> 
> @FloderCC are you using FeedAPI to create new Listing also? if yes, then how are generating forms for input from users based on product type?

Hello. I stopped using FeedAPI to create new Listing when I discovered the Listings Items API in the selling partner api.
Before when I used FeedAPI, products were automatically assigned to the category that Amazon determined. The forms for input was obtained by scrapping the category xsd file.

**Comment by MST1122** [comment]
@FloderCC I am also now testing the new Listing API. But for me difficult part is still generating the Input form. Could you help me with that, how are you doing, and send me some example code.

**Comment by FloderCC** [comment]
> 
> 
> @FloderCC I am also now testing the new Listing API. But for me difficult part is still generating the Input form. Could you help me with that, how are you doing, and send me some example code.

The SP-API defines product types instead of categories. So I use the calls described at [https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/product-type-definitions-api-use-case-guide/definitions-product-types-api-use-case-guide_2020-09-01.md](https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/product-type-definitions-api-use-case-guide/definitions-product-types-api-use-case-guide_2020-09-01.md) to get the data that I need to specify for the product.

**Comment by chapmanjw** [comment]
@FloderCC The `child_parent_sku_relationship` should now be visible in the Product Type Definitions API schemas. Happy listing. :-)

**Comment by alexisbellido** [comment]
> Configuring the variation relationships between SKUs is not yet supported by the Listings Item APIs. This functionality will be enabled in a future release of the APIs. We will post an announcement when this functionality is available.

Hey, @chapmanjw , is this still the case?

I've created a listing (the parent) with two variations (the children) via Seller Central and I can get them all with the getListingsItem operation and I see a few properties in the two children that I think are related to variations: parentage_level, child_parent_sku_relationship, variation_theme.

Now I want to make one of the children an independent listing and I thought I could just call patchListingsItem for that child listing and remove parentage_level, child_parent_sku_relationship, variation_theme but it's not working.

I also tried passing empty lists and empty strings for those properties but still no luck.

I once got this error: "Invalid empty value provided in patch at index of 0.".

What would be the correct way to call the patchListingsItem operation to make a variation (a child) an independent listing?

Thank you.
